cmd/vc-rest/startcmd/start.go

```diff
@@ -103,46 +117,12 @@ func createStartCmd(srv server) *cobra.Command {
 		Short: "Start vc-rest",
 		Long:  "Start vc-rest inside the edge-service",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			hostURL, err := cmdutils.GetUserSetVar(cmd, hostURLFlagName, hostURLEnvKey, false)
-			if err != nil {
-				return err
-			}
-
-			edvURL, err := cmdutils.GetUserSetVar(cmd, edvURLFlagName, edvURLEnvKey, false)
-			if err != nil {
-				return err
-			}
-
-			blocDomain, err := cmdutils.GetUserSetVar(cmd, blocDomainFlagName, blocDomainEnvKey, false)
-			if err != nil {
-				return err
-			}
-
-			hostURLExternal, err := cmdutils.GetUserSetVar(cmd, hostURLExternalFlagName,
-				hostURLExternalEnvKey, true)
-			if err != nil {
-				return err
-			}
-
-			universalResolverURL, err := cmdutils.GetUserSetVar(cmd, universalResolverURLFlagName,
-				universalResolverURLEnvKey, true)
-			if err != nil {
-				return err
-			}
-
-			mode, err := cmdutils.GetUserSetVar(cmd, modeFlagName, modeEnvKey, true)
+			hostURL, edvURL, blocDomain, hostURLExternal, universalResolverURL, mode, localStorePath, err :=
```

Comment:
consider creating a struct with the returned fields and return this struct instead

Reply:
Oh, yes you're right. In fact, there's already a struct called vcRestParameters that's a perfect fit already. In hindsight it seems obvious...not sure why I missed that before. I've updated the code.